disposable_email_blacklist.conf

```diff
@@ -79,10 +79,12 @@ a-bc.net
 a.mailcker.com
 a.vztc.com
 a45.in
+aaathats3as.com
 abyssmail.com
 afrobacon.com
 ag.us.to
 agedmail.com
+airmail.cc
```

Comment:
Found the domain here -- https://cock.li/ -- Haven't had any good traffic from any of their domains, figured I'd include it